.packit.yaml

```diff
@@ -0,0 +1,46 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: ansible-collection-theforeman-foreman.spec
+
+# add or remove files that should be synced
+files_to_sync:
+  - ansible-collection-theforeman-foreman.spec
+  - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: theforeman.foreman
+# downstream (Fedora) RPM package name
+downstream_package_name: ansible-collection-theforeman-foreman
+
+upstream_tag_template: v{version}
+
+actions:
+  post-upstream-clone:
+    - "wget https://github.com/theforeman/foreman-packaging/raw/rpm/develop/packages/plugins/ansible-collection-theforeman-foreman/ansible-collection-theforeman-foreman.spec -O ansible-collection-theforeman-foreman.spec"
+  get-current-version:
+    - python3 -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["version"].replace("-dev", ""))'
+  create-archive:
+    - make dist PYTHON_COMMAND=python3
+    - bash -c "ls -1t ./theforeman-foreman-*.tar.gz | head -n 1"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      rhel-8:
+        additional_modules: "foreman:el8"
+        additional_repos:
+          - https://yum.theforeman.org/releases/nightly/el8/x86_64/
+      rhel-9:
+        additional_repos:
+          - https://yum.theforeman.org/releases/nightly/el9/x86_64/
```

Comment:
technically we don't need those -- keep or drop?

Reply:
drop